### ISSUE TITLE
Steam module messages throttled

### ISSUE THREAD

**Issue by geominorai** [Impact123/CallAdmin#27]
The interface the MessageBot extension has been using seems to have been throttled by Valve significantly now and messages only get sent to 1-2 people and the rest never get it.

**Comment by Impact123** [comment]
Yeah, we're already onto it on it as you can see in [the latest commits to MessageBot](https://github.com/dordnung/MessageBot/commits/master). We found out that [limited accounts](https://support.steampowered.com/kb_article.php?ref=3330-IAGK-7663) hit that limit way sooner. Upgrading it is advised if yours is limited. There's no new release yet, but we try to get it out asap, there will be post [on the forums](https://forums.alliedmods.net/showthread.php?t=213670) when it happens. Thanks for reporting!.

**Comment by geominorai** [comment]
That's awesome.  I'll be looking forward to it.  :+1:
